src/app/utils/tests/MGPValidation.spec.ts

```diff
@@ -8,6 +8,18 @@ describe('MGPValidation', () => {
         const expectedError: string = 'MGPValidation: Cannot extract failure reason from success.';
         expect(() => MGPValidation.SUCCESS.getReason()).toThrowError(expectedError);
     });
+    describe('ofFallible', () => {
+        it('should return MGPValidation.SUCCESS when given a success', () => {
+            // Given a successful MGPFallible
+            const fallible: MGPFallible<number> = MGPFallible.success(42);
+
+            // when converting it to a MGPValidation
```

Comment:
When :p

Reply:
nice, tout not GWT sont en GWT et plus en gwt (maintenant) !